CHANGELOG.rst

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 1.14.2-dev
 ==========
 
+* enhancement: add content_type parameter to sagemaker.tensorflow.serving.Predictor
```

Comment:
nitpick: I'd make `content_type` and `sagemaker.tensorflow.serving.Predictor` monospace

Reply:
ok